sdk/provisioning/Azure.Provisioning.Storage/api/Azure.Provisioning.Storage.net8.0.cs

```diff
@@ -781,7 +781,6 @@ public partial class QueueService : Azure.Provisioning.Primitives.ProvisionableR
         public QueueService(string bicepIdentifier, string? resourceVersion = null) : base (default(string), default(Azure.Core.ResourceType), default(string)) { }
         public Azure.Provisioning.BicepList<Azure.Provisioning.Storage.StorageCorsRule> CorsRules { get { throw null; } set { } }
         public Azure.Provisioning.BicepValue<Azure.Core.ResourceIdentifier> Id { get { throw null; } }
-        public Azure.Provisioning.BicepValue<string> Name { get { throw null; } }
```

Comment:
This member can't be deleted this, it's a part of the public API surface

Reply:
this was added back